mongodb/spring-cloud-starter-stream-source-mongodb/src/main/java/org/springframework/cloud/stream/app/mongodb/source/MongoDBSourceProperties.java

```diff
@@ -0,0 +1,67 @@
+/*
+ * Copyright (c) 2016 the original author or authors.
```

Comment:
No reason to comment if you agree with me.
Just make an appropriate change and comment it in the commit message is enough for me.
The GH has met our prayers recently and now it notifies us if a new commit has been added.

More over I don't see the mention change from AmazonS3 Source. And worse: I was forced to take a look to that code to be sure which Copyright I did there.
So, this comment is pointless and even harmful.
:smile: